backend/lib/services/shoots.js

```diff
@@ -416,14 +422,23 @@ async function getKubeconfigGardenlogin (client, shoot) {
   const { namespace, name } = shoot.metadata
 
   const [
-    ca,
-    gardenClusterIdentity
-  ] = await Promise.all([
-    client.core.secrets.get(namespace, `${name}.ca-cluster`),
+    { value: configmap, reason: configmapError },
```

Comment:
Define the function `getGardenClusterCaData` here, export this function and use it in the terminals code. 
```js
async function getGardenClusterCaData (client, { namespace, name }) {
  try {
    const configmap = await client.core.configmaps.get(namespace, `${name}.ca-cluster`)
    return encodeBase64(configmap.data?.['ca.crt'])
  } catch (err) {
    // TODO(petersutter): Remove this fallback of reading the `<shoot-name>.ca-cluster` Secret when Gardener no longer reconciles it, presumably with Gardener v1.97.
    if (isHttpError(err, 404)) {
      const caCluster = await client.core.secrets.get(namespace, `${name}.ca-cluster`)
      return caCluster.data['ca.crt']
    } else {
      throw err
    }
  }
}
exports.getGardenClusterCaData = getGardenClusterCaData
```

Reply:
✅